ui/imports/shared/stores/NetworkConnectionStore.qml

```diff
@@ -29,9 +29,7 @@ QtObject {
     readonly property var blockchainNetworksDown: root.allBlockchainNetworksDown.filter(chainId => ModelUtils.contains(root.networksStore.activeNetworks, "chainId", chainId))
     readonly property bool atleastOneBlockchainNetworkAvailable: root.blockchainNetworksDown.length < root.activeNetworksCount
 
-    readonly property bool sendBuyBridgeEnabled: localAppSettings.testEnvironment || (isOnline &&
-                                        (!networkConnectionModule.blockchainNetworkConnection.completelyDown && atleastOneBlockchainNetworkAvailable) &&
-                                        !networkConnectionModule.marketValuesNetworkConnection.completelyDown)
+    readonly property bool sendBuyBridgeEnabled: localAppSettings.testEnvironment || (isOnline && !networkConnectionModule.blockchainNetworkConnection.completelyDown && atleastOneBlockchainNetworkAvailable)
     readonly property string sendBuyBridgeToolTipText: !isOnline ? qsTr("Requires internet connection") :
```

Comment:
This one also requires an update, otherwise we will see an incorrect tooltip for the Send button.

Reply:
@saledjenic, thanks, did it (and simplified a bit)